decidim-proposals/config/locales/en.yml

```diff
@@ -130,6 +130,7 @@ en:
             votes: Votes
     resource_links:
       included_proposals:
+        results: 'Proposal included in these projects:'
```

Comment:
@oriolgual I think we should refactor these translations so something like:

```yaml
resource_links:
  included_proposals:
    from: %{resource} includes these %{resources}
    to: %{resource} included in these %{resources}
```

This way we don't depend on every possible combination of resources, just what the association `means`.